Removed spurious collectd reference.

claytono opened the pull request:
All of the collectd/nagios/graphite support is actually in the `coi` module, and this appears to be completely unnecessary.  This prevents users from not installing collectd even if they've set the `enable_monitoring` hiera key to false.

claytono committed:
Removed spurious collectd reference.

markvoelker commented:
This looks good to me--could you target this to the Icehouse branch and then cherry pick back to havana though?

claytono committed:
Remove proxy parameters that were never pushed upstream

claytono committed:
Added option to not configure provider networker int

This allows disabling the provider network interface configuration, so
that it can be configured via other means.

markvoelker commented:
This hasn't been retargetted by the original submitter in a couple of weeks, so I've taken the liberty:

https://github.com/CiscoSystems/puppet-coe/pull/49
